src/content/tutorial/1-basics/1-introduction/3-one-way-data-flow/_files/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Signals</title>
+
+  <link rel="stylesheet" href="/style.css" />
+</head>
+
+<body>
+  <div id="app">
```

Comment:
since we're defining the starting HTML, we could omit the "delete this" from the content.md?

Reply:
That makes sense, the reasoning was to walk through from a working imperative example to a declarative one. Instead it could just focus on implementing this example, is that right? In that case we'd delete the original JS as well right, and the text that references it? At that point I suppose we could initialize the JS with the `state` and `element` declarations and imports, and then the user would add the render function and DOM manipulation?

Separately (and maybe the above changes make room for this), I'm thinking if we want to better illustrate "top-down data flow" it would make sense to have a `Counter` function component be called by `createUI`. And then possibly rename `createUI` to `App`? I named it to correspond to the `ui = f(state)` but changing might be clearer.

Reply:
yeah, I think that plan sounds awesome! :tada: